src/filepattern/functions.py

```diff
@@ -14,8 +14,8 @@ def get_regex(filepattern: str, suppress_warnings=False) -> tuple:
         String containing the regex equivalent of the filepattern
 
     """
-    result = backend.FilePattern.getRegex(filepattern, suppress_warnings)
-    return result[0:2]
```

Comment:
So, this was a bug not feature, right?

Reply:
I added this functionality to the C++ side with the getRegex function I added to the bindings. It now only returns the string version of the regex and not the entire tuple that contained the named groups like it did before. The bug was a missing binding for the getRegex function